packages/drift_sqlite_async/lib/src/connection.dart

```diff
@@ -15,12 +15,22 @@ import 'package:sqlite_async/sqlite_async.dart';
 class SqliteAsyncDriftConnection extends DatabaseConnection {
   late StreamSubscription _updateSubscription;
 
-  SqliteAsyncDriftConnection(SqliteConnection db, {bool logStatements = false})
-      : super(SqliteAsyncQueryExecutor(db, logStatements: logStatements)) {
+  SqliteAsyncDriftConnection(
+    SqliteConnection db, {
+    bool logStatements = false,
+    Set<TableUpdate> Function(UpdateNotification)? transformTableUpdates,
+  }) : super(SqliteAsyncQueryExecutor(db, logStatements: logStatements)) {
     _updateSubscription = (db as SqliteQueries).updates!.listen((event) {
-      var setUpdates = <TableUpdate>{};
-      for (var tableName in event.tables) {
-        setUpdates.add(TableUpdate(tableName));
+      final Set<TableUpdate> setUpdates;
+      // This is useful to map local table names from PowerSync that are backed by a view name
+      // which is the entity that the user interacts with.
```

Comment:
Can we make this a dartdoc comment on the constructor itself?

It would also be good to have a small unit test for this scenario - I'd be happy to help with that too.

Reply:
Updated docs and we've added a new test